spec/services/net_ssh_agent_service_spec.rb

```diff
@@ -0,0 +1,30 @@
+describe NetSshAgentService do
+  before(:all) do
```

Comment:
Avoid using before(:all) and after(:all).  Each test should be completely independent.